guidelines/releases/release-communication.md

```diff
@@ -0,0 +1,36 @@
+# Release communication
+
+At the beginning of the release process, the team responsible for the release picks a Release Master who will manage the release process. One of the Release Master's responsibilities is to update the rest of the organization on the progress of the release process. The Release Master is obliged to do the following:
+
+- Inform Kyma teams about release deadlines on Slack channels at least one week before the first release candidate (RC1), and pin the message so that it is easy to find. 
+
+- Prepare the environment to test a release candidate. Share access to the testing environment with Kyma teams so that they can test their components before merging the RC.
+
+- Create an excel sheet where all the manual tests are defined and ensure that all tests are executed by responsible teams.
+
+- Inform Kyma teams about deadlines for cherry-picking changes to the release branch.
+
+- Inform Kyma teams about all the release-related pull requests that require approvals.
+
+- Notify Kyma teams about failing tests to fix them as quickly as possible. Kyma teams responsible for failing tests should communicate with a Release Master about the fixing progress.
+
+## Communication channels
+
+The Release Master communicates the release progress simultaneously on these two internal Slack channels: 
+
+- `c4core-xf-team` - the main channel to communicate with the Kyma teams
+- `c4core-kyma-scrum-masters` - a channel used to communicate the release progress to Scrum Masters
```

Comment:
Those are private channels. I think we shouldn't mention them here.

Reply:
I deleted the information about them